generator/vertx/src/main/resources/META-INF/fabric8/generator-default

```diff
@@ -1 +1 @@
-io.fabric8.maven.generator.vertx.VertxGenerator,1000
+io.fabric8.maven.generator.vertx.VertxGenerator
```

Comment:
btw, fixed the issue that genertors having the same order overwrite each other, so that some get lost. The problem was that they returned 0 in `compareTo()` which is used in `TreeSet` also for equals(). See #722 for details.